src/styles/utils/variables.scss

```diff
@@ -0,0 +1,8 @@
+$headerPadding: 30px;
+$DarkColor: #1d1d1d;
+$titleSizeIpad: 48px;
+$titleSizeDesktop: 64px;
+$textLineHeight: 140%;
+$DarkGrayColor: #292929;
```

Comment:
names for colors should follow this convention:

$color-dark-gray;
$color-white;
$color-light-gray;

it's shorter and easier to read :)